src/ExecutionPathTracer/ExecutionPathTracer.csproj

```diff
@@ -32,7 +32,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="..\..\build\assets\qdk-nuget-icon.png" Pack="true" Visible="false" PackagePath="" />
+    <PackageReference Include="Microsoft.Quantum.Simulators" Version="0.12.20072811-beta" />
```

Comment:
Looks like a bad merge here, I think you meant to keep the line with the icon?
```
<None Include="..\..\build\assets\qdk-nuget-icon.png" Pack="true" Visible="false" PackagePath="" />
```

Reply:
That was a merge issue, thanks for noticing that. Will fix.

Reply:
Thanks! And just to clarify, the "bad" in my comment was referring to the suggestion from the merge tool, not to the user of the merge tool... :-)